### ISSUE TITLE
Sherlock wrapper cache needs a process for invalidating old entries

### ISSUE THREAD

**Issue by gpfrancis** [lsst-uk/lasair-lsst#79]

**Comment by thespacedoctor** [comment]
Hi @gpfrancis  - I'm not sure what is being asked for here? Is this something I need to deal with?

**Comment by gpfrancis** [comment]
@thespacedoctor - The issue here is that we should not annotate alerts with cached information that is out of date because a new version of Sherlock/catalogue/crossmatch algorithm is now available. Unfortuntaely, the only information that we have available at present is when Sherlock was run and what version. With this we could use a TTL on the cache, but that is inefficient and/or slow to update, or we could use the Sherlock version and hope that nothing else is ever changed without bumping the version number as well. I spoke to @genghisken about this and he told me that there is a wider issue around this with regard to recording prevenance that is already being looked at and hopefully that will give us a solution to this problem as well. Given this we decided to wait on the outcome of that discussion before doing anything else about this.

As far as I know you don't need to do anything new that you're not already doing.

**Comment by thespacedoctor** [comment]
As of v2.2.0 Sherlock algorithm now ships alongside the code (and is therefore version controlled)

**Comment by tms-epcc** [comment]
As discussed with @gpfrancis via email this is not a priority in Lasair cycle 5 and hence is moved into the backlog.
